frontend/website-redesign/src/pages/Testnet.re

```diff
@@ -0,0 +1,166 @@
+module Styles = {
+  open Css;
+
+  let page =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      overflowX(`hidden),
+      height(`percent(100.)),
+      height(`percent(100.)),
+      selector("h4", [important(color(Theme.Colors.white))]),
+    ]);
+
+  let statusBadge =
+    merge([
+      Theme.Type.label,
+      style([color(Theme.Colors.white), marginTop(`rem(4.5))]),
+    ]);
+
+  let leaderboardLink = style([textDecoration(`none)]);
+
+  let leaderboardTextContainer =
+    style([
+      display(`flex),
+      flexDirection(`column),
+      alignItems(`center),
+      justifyContent(`center),
+      marginTop(`rem(4.)),
+      marginBottom(`rem(2.)),
+      width(`percent(100.)),
+      media(
+        Theme.MediaQuery.notMobile,
+        [
+          width(`percent(50.)),
+          alignItems(`flexStart),
+          justifyContent(`flexStart),
+        ],
+      ),
+      selector(
+        "button",
+        [
+          marginTop(`rem(2.)),
+          important(maxWidth(`percent(50.))),
+          width(`percent(100.)),
+        ],
+      ),
+    ]);
+
+  let disclaimer =
+    merge([
+      Theme.Type.disclaimer,
+      style([
+        display(`listItem),
+        listStylePosition(`inside),
+        marginBottom(`rem(14.)),
+        marginTop(`rem(3.)),
+      ]),
+    ]);
+
+  let gradientSectionExpanded =
+    style([
+      height(`auto),
+      width(`percent(100.)),
+      position(`relative),
+      overflow(`hidden),
+      display(`flex),
+      flexWrap(`wrap),
+      marginLeft(`auto),
+      marginRight(`auto),
+      justifyContent(`center),
+    ]);
+
+  let gradientSection =
+    merge([
+      gradientSectionExpanded,
+      style([
+        height(`rem(53.)),
+        after([
+          contentRule(""),
+          position(`absolute),
+          bottom(`px(-1)),
+          left(`zero),
+          height(`rem(8.)),
+          width(`percent(100.)),
+          pointerEvents(`none),
+          backgroundImage(
+            `linearGradient((
+              `deg(0.),
+              [
+                (`zero, Theme.Colors.white),
+                (`percent(100.), Theme.Colors.white),
+              ],
+            )),
+          ),
+        ]),
+      ]),
+    ]);
+};
+
+[@react.component]
+let make = () => {
+  <Page title="Mina Protocol" footerColor=Theme.Colors.orange>
+    <div className=Styles.page>
+      <div className=Nav.Styles.spacer />
+      <Hero
+        title="Testnet"
+        header="Secure the Network"
+        copy={j|"Push the boundaries of Mina’s testnet to help prepare for mainnet."|j}
```

Comment:
delete the quotation marks here, they're showing up in the header and dont need to be!